deadbelly/whats-cookin
# Baf/feature/create and test pantry class

## Pull Request

<br>

### Description

* Is this a feature or a fix?
This is a feature

* Where should the reviewer start?
Open what's cookin' in atom, open pantry.js and pantry-test.js.

* What functionalities do these changes effect? Why were these changes made? What was the motivation behind these changes?
This class can evaluate whether a user has enough ingredients to cook a meal. If the user has enough ingredients then the ingredients will be removed from the user's pantry, and if the user DOESN'T have enough ingredients, it will output an array of ingredients needed to still cook a meal represented as objects with all the same properties, but now including a total cost to acquire enough of the ingredient to cook the meal in question. 

* What outside features or research did you use to implement this fix?

* How should this be tested?
CD into whats cookin, run npm test test/pantry-test.js and check that the tests are passing.
* Applicable screenshot(s): 

<br>

***

#### Please review considerations below:
